x-pack/plugins/lists/server/services/exception_lists/utils/import/import_exception_list_items.ts

```diff
@@ -46,8 +46,14 @@ export const importExceptionListItems = async ({
     // Gather lists referenced by items
     // Dictionary of found lists
     const foundLists = await getAllListTypes(
-      agnosticListItems,
-      nonAgnosticListItems,
+      agnosticListItems.map((list) => ({
```

Comment:
it looks like the function `(list) => ({
        listId: list.list_id,
        namespaceType: list.namespace_type,
      })` is being used a couple of times in this file and import_exception_lists.ts. we could probably assign it to a variable & reuse it like `agnosticListItems.map(mapList)`